quantum-adapters/kiwoom-adapter/src/kiwoom_api/api/websocket.py

```
@@ -0,0 +1,825 @@
+        try:
+            trnm = kiwoom_data.get('trnm', 'UNKNOWN')
+            
+            if trnm == 'REAL':
+                # REAL 메시지 처리
+                data_list = kiwoom_data.get('data', [])
+                converted_items = []
+                
+                for item in data_list:
+                    stock_code = item.get('item', 'N/A')
+                    data_type = item.get('type', 'N/A')
+                    data_name = item.get('name', 'Unknown')
```

Ruff (0.12.2)

Local variable `data_name` is assigned to but never used

Remove assignment to unused variable `data_name`

(F841)